src/tests/server.mjs

```diff
@@ -171,7 +171,6 @@ router.get("/messages", (request, response) => {
 function receiveMessage(content, id, target) {
   const data = renderSSEData(renderMessage(content, id, target))
   for (const response of streamResponses) {
-    console.log("delivering message to stream", response.socket?.remotePort)
```

Comment:
While I was removing all my console.log() statements I found this old one left in the code.